lace/emulator/gp_emulator.py

```diff
@@ -19,57 +20,91 @@ class GPEmulator:
     """
     def __init__(self,basedir="/lace/emulator/sim_suites/Australia20/",
                 p1d_label=None,skewers_label=None,
-                max_archive_size=None,verbose=False,kmax_Mpc=10.0,
-                paramList=None,train=True,drop_tau_rescalings=True,
-                drop_temp_rescalings=True,keep_every_other_rescaling=False,
-                undersample_z=1,emu_type="k_bin",z_max=5,z_list=None,
-                passarchive=None,set_noise_var=1e-3,asymmetric_kernel=True,
-                check_hull=False,set_hyperparams=None,
-                paramLimits=None,rbf_only=True,
+                verbose=False,kmax_Mpc=4.0,
```

Comment:
A bit risky to change the default value of kmax_Mpc here. Probably fine, since we were probably specifying this always, but it might be safer to force the user to provide a value (so that the code would crash if an old script was not specifying it).